heartbeat/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2.1'
+services:
+  beat:
+    build: ${PWD}/.
+    depends_on:
+    - proxy_dep
+    env_file:
+    - ${PWD}/build/test.env
+    environment:
+    - REDIS_HOST=redis
+    - REDIS_PORT=6379
+    - KIBANA_HOST=kibana
+    - KIBANA_PORT=5601
+    working_dir: /go/src/github.com/elastic/beats/heartbeat
+    volumes:
+    - ${PWD}/..:/go/src/github.com/elastic/beats/
+    # We launch docker containers to test docker autodiscover:
+    - /var/run/docker.sock:/var/run/docker.sock
+    command: make
+
+  # This is a proxy used to block beats until all services are healthy.
+  # See: https://github.com/docker/compose/issues/4369
+  proxy_dep:
+    image: busybox
+    depends_on:
+      elasticsearch: { condition: service_healthy }
+      kibana:        { condition: service_healthy }
+      redis:         { condition: service_healthy }
+
+  elasticsearch:
```

Comment:
As the output in the system tests below is not Elasticsearch, do we need elasticsearch?

Reply:
Yep, we can get rid of that as well. should happen in the #8491 merge.